scss/forms/_form-control.scss

```diff
@@ -191,7 +191,24 @@ textarea {
 .form-control-color {
   width: $form-color-width;
   height: $input-height;
-  padding: $input-padding-y;
+  padding: $form-color-padding; // Boosted mod
+  border: 2px solid var(--#{$prefix}body-color);
+
+  // Boosted mod
+  &:focus {
+    &[data-focus-visible-added] {
+      background-color: $black;
+    }
+  }
+
+  &:not(:disabled):not([readonly]):hover {
```

Comment:
I'm not 100% sure but do we need to have such a complex selector since readonly doesn't seem to work on color input and less sure about this one but IMO, `:disabled` should have a `pointer-events: none` so doesn't need to be specified here (would need to add so on a disabled label `.is-disabled`).